geoh5py/objects/drillhole.py

```diff
@@ -61,6 +63,8 @@ def __init__(self, object_type: ObjectType, **kwargs):
         self._deviation_y = None
         self._deviation_z = None
 
+        self._tolerance = 1e-2
```

Comment:
**suggestion (naming, semantic):** `default_colocation_distance` ? or just `default_colocation`

- "default" because this get used only if not specified in `attrs` of `add_data()` (should also rename tolerance to `colocation` in those `attrs`?)

Reply:
Done